doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,277 @@
+
+If you haven't already done so, you'll need to create a (free) GitHub account on http://github.com and have
+Git tools available on your development system.  (For Windows users you can use `Git Bash`_ or other Git command line tools.)
+
+.. _`Git Bash`: https://git-for-windows.github.io/
+
+Prepare your patch
+------------------
+
+.. _`Fork a Repo`: https://help.github.com/articles/fork-a-repo/
+
+#. Create a Fork (using GitHub's web interface)
+   In GitHub, create a fork of the repo containing the layer you need to modify. In this example we use ``meta-ostro``. 
+   For more information, see GitHub's `Fork a Repo`_ help page.
+
+   In your web browser, navigate to the repo: https://github.com/ostroproject/meta-ostro and click on the Fork button
+   in the top right corner to fork your own copy of the ``ostroproject/meta-ostro`` repo to your account.
+
+#. Create a repository on your local computer to your fork.  If you have ssh keys generated you can register your 
+   public key on your GitHub account (SSH and GPG keys in your Personal Settings on Git Hub) to be authorized.  
+   Otherwise, you can clone with "https" and specify your GitHub username and password:::
+ 
+      $ git clone github.com:<your-username>/meta-ostro                # if you've registered your ssh key
+      $ git clone https://github.com/<your-username>/meta-ostro.git    # if not use this (and your GitHub username/password)
+
+      $ cd meta-iot-os
+
+      $ git remote add upstream github.com:ostroproject/meta-ostro            # if you've registered your ssh key
+      $ git remote add upstream https://github.com/ostroproject/meta-ostro    # if you cloned with https
+
+      $ git remote -v        # verify origin (your fork) and remote (Ostro OS master) are defined as expected
+
+#. Create a new branch to work on your patch:::
+
+      $ git fetch upstream
+      $ git checkout -b my-patched-branch upstream/master
+
+#. Make and test your changes
+   After making your edits or adding files, this typically involves building a new image that contains your changes. 
+
+   To replace the current layer with the one you are working on, modify your <builddir>/conf/bblayers.conf file as appropriate
+   and then run bitbake to start the build:::
+
+      $ bitbake -k ostro-image-noswupd        # for example, other target images are available
+
+#. Commit your changes and rebase onto master
+   After you’ve tested and verified your change does what was intended, you can commit your change locally. 
+   Make sure that you follow the `Code Review Process Guidelines`_ described earlier in this document:::
+
+      $ git commit -a -s          # follow guidelines for the commit message
+
+   Depending on how long you have worked on your patch, it may be that the master branch has evolved since you branched 
+   it off. If that is the case, you should rebase your working branch onto master before sending the Pull Request (PR):::
+
+
+      $ git rebase upstream/master
+
```

Comment:
We're missing one step here which explains how to push your branch to your forked repo.
$ git push origin my-patched-branch